server/commands/documentUpdater.ts

```diff
@@ -74,7 +77,9 @@ export default async function documentUpdater({
   const changed = document.changed();
 
   if (publish) {
-    document.lastModifiedById = user.id;
+    if (document.isDraftWithoutCollection) {
```

Comment:
I feel like draft is increasingly meaningless in this world and we can probably lose this method in favor of just checking if `collectionId` is not set and have the same behavior.